src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -125,6 +158,40 @@ public function tolerates(array $deprecationGroups)
         return true;
     }
 
+    /**
+     * @return bool
+     */
+    public function isBaselineDeprecation(Deprecation $deprecation)
+    {
+        $result = \in_array($deprecation->getMessage(), $this->baselineDeprecations, true);
```

Comment:
So, we'll ignore any new occurence for a deprecation message.
Shouldn't we account for the number of occurrences for the baseline? As well as the class::method / triggering file?

The way this feature works in PHPStan is accounting at least for occurrences per class: https://medium.com/@ondrejmirtes/phpstans-baseline-feature-lets-you-hold-new-code-to-a-higher-standard-e77d815a5dff

Reply:
I agree with @ogizanagi, checking the count would be great.